selfdrive/ui/ui.cc

```diff
@@ -255,7 +257,9 @@ Device::Device(QObject *parent) : brightness_filter(BACKLIGHT_OFFROAD, BACKLIGHT
 }
 
 void Device::update(const UIState &s) {
-  updateBrightness(s);
+  if (s.sm->frame % 2 == 0) {
```

Comment:
Why was the frequency changed? This messes with the `brightness_filter` time constant.